docs/en/operations/settings/index.md

```diff
@@ -13,13 +13,13 @@ XML-based Settings Profiles and [configuration files](https://clickhouse.com/doc
 There are two main groups of ClickHouse settings:
 
 - Global server settings
-- Query-level settings
+- Session settings
```

Comment:
We already have the session settings and they are not the same as query settings, they are set differently:

- session setting:
```sql
SET setting1 = value1;
```

- query setting:
```sql
SELECT * FROM mytable SETTINGS setting1 = value1;
```

Reply:
True, yet "session" settings and "query settings" are conceptually in the same "group" (l. 13) - one can override the other. The other group are global settings, so I think my changes here still make sense.

Reply:
> "session" settings and "query settings" are conceptually in the same "group"

Yes, they are in the same group, but how are we going to call this group? Because if we say that session settings and query settings are in the same group and that group is also named session settings then that doesn't sound very good.

Reply:
Fair enough. I would argue the main point is how session/query setting are _primarily_ used. We can say (and actually, l. 18 does say) that session settings apply to _sessions_ BUT they can also apply to queries. The latter is supposedly an exception, not the rule. (I think we as devs use session settings mostly per-query but normal users won't).

The other point is that the other name "core settings" is ambiguous as well. Imagine one group is called "global settings" and the other group "core settings" - to me these terms seem to overlap ...